bundler/spec/other/major_deprecation_spec.rb

```diff
@@ -573,8 +573,6 @@
       before do
         bundle :show
       end
-
```

Comment:
@NeimadTL I just noticed that actually, all specs inside this "`context "bundle show" do` block, except for the one about the `--outdated` flag can now be removed. Can you do it?

Reply:
Yes, it's correct, thanks!